packages/block-editor/src/components/iframe/index.js

```diff
@@ -221,11 +222,32 @@ function Iframe( {
 
 	const [ iframeWindowInnerHeight, setIframeWindowInnerHeight ] = useState();
 
+	const windowResizeRef = useRefEffect( ( node ) => {
+		const nodeWindow = node.ownerDocument.defaultView;
+
+		setWindowInnerWidth( nodeWindow.innerWidth );
+		const onResize = () => {
+			setWindowInnerWidth( nodeWindow.innerWidth );
+		};
+		nodeWindow.addEventListener( 'resize', onResize );
+		return () => {
+			nodeWindow.removeEventListener( 'resize', onResize );
+		};
+	}, [] );
+
+	const [ windowInnerWidth, setWindowInnerWidth ] = useState();
+
+	const scale = useMemo( () => {
+		return typeof scaleProp === 'function'
+			? scaleProp( {
+					containerWidth,
+					windowInnerWidth,
+			  } )
```

Comment:
The signature of this function changed, but the old function didn't make it into 6.5, so I think it's okay, but it might be a good idea to make the scale as a function a private API before the 6.6 release. We might want to move some of the logic elsewhere.